docs/tutorial/part03.rst

```diff
@@ -2,5 +2,245 @@
+
+    ├── fetch_api/
+    │   ├── __init__.py
+    │   ├── admin.py
+    │   ├── apps.py
+    |   ├── models.py
+    │   ├── migrations/
+    │   │   └── __init__.py 
+
+
+Create The Models Directory:
+==============================
+
+On The ``fetch_api`` folder, first, we need to remove the ``models.py`` file. Then we need to create a new folder called ``models``.  Inside the folder, you will need to add the files with the structure of the nodes. Each file will have the essential fields and methods of the data stored in Neo4j database. 
+
+.. [*] Note: Before we start creating our models we need to create a  ``__init__.py`` file inside the model’s folder so Python treat the directory as a module
+
+Start creating The Models Files:
+===================================
+To start we will create an ``entity.py`` file: 
+
+1. The first step is to add the imports needed::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipTo,
+        RelationshipFrom
+    )
+
+2. Create a class for the node call Entity. We are connecting to a Neo4j database instead of the regular Django database; therefore,  we are going to use  ``StructuredNode``.  This is the equivalent of  ``models.Model`` which is usually  used when creating a Models class in Django. When using ``StructuredNode`` , neomodel automatically creates a label for each class using it with the corresponding indexes and constraints.
+
+ If you need ``ModelForm``, you will need to change ``StructuredNode`` to ``DjangoNode``. Also, you will need to add a ‘Meta’ class. For more reference see the documentation for django-neomodel_
+
+    .. _django-neomodel: https://github.com/neo4j-contrib/django-neomodel
+
+    ``class Entity(StructuredNode):``
+
+
+3. The next step is to add the properties for the node. Each node property in the Neo4j database should be a property in the model class.
+
+The properties of  the ``Entity`` node in the database have the following scheme::
+
+        {
+          "sourceID": "Panama Papers",
+          "address": "MEI SERVICES LIMITED ROOM E; 6TH FLOOR; EASTERN COMMERCIAL CENTRE; 395-399 HENNESSY ROAD HONG KONG",
+          "jurisdiction": "SAM",
+          "service_provider": "Mossack Fonseca",
+          "countries": "Hong Kong",
+          "jurisdiction_description": "Samoa",
+          "valid_until": "The Panama Papers data is current through 2015",
+          "ibcRUC": "25475",
+          "name": "JIE LUN INVESTMENT LIMITED",
+          "country_codes": "HKG",
+          "incorporation_date": "10-APR-2006",
+          "node_id": "10000020",
+          "status": "Active,
+        }
+
+Therefore, the structure of the ``Entity`` class should be::
+
+    class Entity(StructuredNode):
+        sourceID                           = StringProperty()

```

Comment:
I think we should talk about Property Types around here. Also,  in this case, explain the reason you only use StringProperty type.

Reply:
good idea :)